Hexastore.Web/EventHubs/EventReceiver.cs

```diff
@@ -105,7 +105,7 @@ public Task ProcessEventsAsync(StoreEvent storeEvent)
                         _storeProcessor.PatchJson(storeId, data);
                         break;
                     case EventType.PATCH_TRIPLE:
-                        var patch = JObject.Parse(storeEvent.Data);
+                        var patch = (JObject)storeEvent.Data;
```

Comment:
nit: This should already be JObject

Reply:
Just saw StoreEvent object - perhaps that can be a JObject now.

Reply:
There is one spot where StoreEvent.Data takes a JArray directly: https://github.com/angshuman/hexadb/blob/304eb874d67f9813017a79fcd6859a3f40c21a48/Hexastore.Web/Controllers/StoreController.cs#L118

If we fix that it could be a JObject.